get_databricks_cluster_eventlog.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Default directory to save results to
+SCRIPT_DIR="$(dirname "$0")"
+results_dir="$SCRIPT_DIR/databricks_cluster_eventlogs"
+mkdir -p $results_dir
+
+function print_usage() {
+  cat <<-EOD
+  Usage: $0 -c <clusterId> [-r <results directory>]
+  Options:
+  -c <cluster ID>  Databricks cluster ID
+  -r <results directory>  directory in which to save cluster eventlog
+  -h                      show this helpful message and exit
+EOD
```

Comment:
The formatting's just a bit off with the description for "-c" not lined up with the other descriptions

Reply:
The "-" in "<<-" isn't necessary because there's no tab characters